test/system/280-update.bats

```diff
@@ -127,4 +127,27 @@ device-write-iops   = /dev/zero:4000 | - | -
     fi
 }
 
+@test "podman update - set restart policy" {
+    TMPD=$(mktemp -d podman-update-test.restart.XXXXXXXX)
+    touch ${TMPD}/sentinel
+    run_podman run --name testctr -v ${TMPD}:/testdir -d alpine sh -c "touch /testdir/alive; while test -e /testdir/sentinel; do sleep 0.1; done;"
```

Comment:
your volume is missing `:Z` so it likely cannot write the dir thus failing the check